platform/cosmos/api.go

```diff
@@ -158,8 +158,10 @@ func fillDelegate(tx *blockatlas.Tx, delegate MessageValueDelegate, msgType stri
 	title := blockatlas.KeyTitle("")
 	switch msgType {
 	case MsgDelegate:
+		tx.Direction = blockatlas.DirectionOutgoing
 		title = blockatlas.AnyActionDelegation
```

Comment:
can you also add this: https://github.com/trustwallet/blockatlas/issues/617, should be a quick one

Reply:
this is not so easy because we need to filter 2 log messages, I going to do that in another PR. And keep this PR simple